src/insights_client/tests/requirements.txt

```diff
@@ -1,5 +1 @@
-pytest~=4.6.11
-mock~=3.0.5
-requests~=2.25.1
-six~=1.15.0
```

Comment:
`six` is still needed, used both in insights-client itself and in one test (`test_sed`)